src/constants.js

```diff
@@ -41,4 +41,7 @@ export const PROGRAMME_PATH = '/utdanning';
 export const PROGRAMME_PAGE_PATH = '/utdanning/:programme';
 
 export const MULTIDISCIPLINARY_SUBJECT_PAGE_PATH =
-  '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7';
+  '/subject:d1fe9d0a-a54d-49db-a4c2-fd5463a7c9e7/:topicPath*/';
```

Comment:
Vil denne treffe på forsida på subjectet, eller skulle du hatt med en ? for å vise at topicPath kan være tom?

Reply:
Yes, den matcher om ingen topic er tilstede også :+1: